client/shared/ButterToast.re

```diff
@@ -69,7 +69,7 @@ module ToastOption = {
 
   [@bs.deriving abstract]
   type t = {
-    content: contentArgs => React.reactElement,
+    content: contentArgs => ReactOld.reactElement,
```

Comment:
Using `ReactOld` temporarily is ok while you work on this, but ultimately it's not needed and `ReactOld` should be removed before we merge the jsx3 migration.

Reply:
Removed `ReacOld`